networking/synthetic/config.yaml

```diff
@@ -1,4 +1,5 @@
 ---
 environment:
-  master: master.example.com
-  nodes: node1.example.com node2.example.com
\ No newline at end of file
+  master: ip-172-31-22-251.us-west-2.compute.internal
+  nodes: ip-172-31-13-12.us-west-2.compute.internal ip-172-31-33-87.us-west-2.compute.internal
+  version: 3.5
```

Comment:
@mffiedler  I realized this after the commit so corrected in the second commit with the comment - "new config file"